test/test_netlify.rb

```diff
@@ -92,6 +93,38 @@ class Jekyll::NetlifyTest < Minitest::Test
       end
     end
   end
+  context 'netlify deploy-preview context' do
+    setup do
+      Jekyll.instance_variable_set(
+        :@logger, Jekyll::LogAdapter.new(Jekyll::Stevenson.new, :error)
+      )
+
+      ENV.clear
+      ENV['CONTEXT'] = 'deploy-preview'
+      ENV['JEKYLL_ENV'] = 'staging'
+
+      ENV['PULL_REQUEST'] = 'false'
+      ENV['DEPLOY_URL'] = 'https://578ab634d5d5cf960d620--open-api.netlify.com'
+      ENV['DEPLOY_PRIME_URL'] = 'https://beta--open-api.netlify.com'
+
+      config = Jekyll.configuration(
+        source: jekyll_test_site,
+        destination: File.join(jekyll_test_site, '_site'),
+      )
+      @site = Jekyll::Site.new(config)
+      @site.read
+      @site.generate
+    end
+
+    context 'info' do
+      setup do
+        @netlify = @site.config['netlify']
+      end
+      should 'be staging-deploy-preview' do
+        assert_equal('staging-deploy-preview', @netlify['environment'])
```

Comment:
also check `@site.config['environment']` here

Reply:
and here you are using brackets, but elsewhere no brackets.

I dont mind which style we use.  Do you have a preference?